### DISCUSSION TITLE
problem with clipPath of group

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8700] [General]
hi,everyone,i want to ask what's the difference that add object to group between by added after new group and initialize?
https://codesandbox.io/s/border-problem-of-group-wmh0xp?file=/src/App.vue
one the hand:
`var rect = new fabric.Rect({});var group= new fabric.Group([rect ],{})`
![image](https://user-images.githubusercontent.com/41353266/219302240-9b0a340d-65f2-4558-a151-b97fcfad7904.png)

one the other hand:
`var rect = new fabric.Rect({});var group= new fabric.Group([],{});group.add(rect)`
![image](https://user-images.githubusercontent.com/41353266/219302575-b6aa7604-dba7-420b-94ea-62420dada5df.png)

when i added clipPath for group.i must set originX to center and originY to center.if i havenot added anyone to group,the layout of group is wrong.but when i added one object to group initialized.the layout of group is ok.

**Comment by ShaMan123** [comment]
This is a bug

**Comment by asturur** [comment]
let's convert it to issue, but let give me time to read it. Interactions between those new layouts and clippath are still hard to grasp for me.
We need to decide if this needs to be fixed or if there is a specific way to do this and nothing to fix

**Reply by ShaMan123** [reply]
I did that
#8703

**Reply by ShaMan123** [reply]
It is a bug, not big
I found a solution to workaround and how to fix
